Remotion/Validation/UnitTests/Implementation/DelegateBasedValidationMessageTest.cs

```diff
@@ -15,29 +15,55 @@
 // along with re-motion; if not, see http://www.gnu.org/licenses.
 // 
 using System;
+using System.Globalization;
 using NUnit.Framework;
+using Remotion.Validation.Implementation;
 
 namespace Remotion.Validation.UnitTests.Implementation
 {
   [TestFixture]
   public class DelegateBasedValidationMessageTest
   {
     [Test]
-    [Ignore("RM-5906")]
-    public void Format_UsesSuppliedFormatProviderInsteadOfCurrentCulture ()
+    [SetCulture("en-US")]
+    [SetUICulture("de-CH")]
+    public void Format_WithFormatProvider_UsesSuppliedFormatProviderInsteadOfCurrentCulture ()
     {
+      Func<string> validationMessageProvider = () => "{0} : {1}";
+      var validationMessage = new DelegateBasedValidationMessage(validationMessageProvider);
+      var culture = CultureInfo.GetCultureInfo("fr-FR");
```

Comment:
```suggestion
      var culture = CultureInfo.GetCultureInfo("en-US");
```
Here, we test for the formatprovider, so we we want to ensure that uiculture  and ambient culture are both different enough from the formatprovider-culture to notice if we use the wrong one. we don't care if the ambient culture or the uiculture can be distiguished in this test because no resources are looked up so the uiculture us not relevant as long as it's different.